### PR TITLE
Bumped dapr/dapr in cert tests

### PR DESCRIPTION
**Pull request by ItalyPaleAle**
SHould make the cert tests work again after the interface for pubsub has changed

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -358,6 +358,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+replace go.opentelemetry.io/otel => go.opentelemetry.io/otel v0.20.0
```
Comment (ItalyPaleAle) [inline go.mod:361]: Explained: https://github.com/dapr/components-contrib/pull/1773

**Review by yaron2 — APPROVED** [review]

**Comment by yaron2** [comment]
@ItalyPaleAle let me know when you're comfortable to merge.

**Comment by ItalyPaleAle** [comment]
@yaron2 I just pushed another commit as when the PR was merged, a go.mod file was accidentally deleted.

There are still 3 failing tests:

- certification for Event Hubs -> that test is broken so it's ok
- certification for Kafka (pubsub and binding) -> those are flaky and are passing ~50% of times. @skyao is looking into those
- conformance for Event Hubs -> investigating this

Fixing the last one will be a hard requirement for me.

However I understand this PR is pretty urgent either, so if you want to go ahead and merge it, I can make a follow-up to fix the Event Hubs conformance test.

**Review by artursouza — APPROVED** [review]

**Comment by ItalyPaleAle** [comment]
I fixed the issue with Event Hubs

**Review by berndverst — APPROVED** [review]

**Pull request merged**